integration/client/client_test.go

```diff
@@ -188,7 +189,9 @@ func TestMain(m *testing.M) {
 				fmt.Fprintln(os.Stderr, "failed to wait for containerd", err)
 			}
 		}
-
+		if err := syscall.Unmount(defaultRoot, syscall.MNT_FORCE); err != nil {
+			fmt.Fprintln(os.Stderr, "failed to  unmount test root dir", err)
+		}
```

Comment:
Is defaultRoot mounted at this time? Wouldn't it log this error most of the time?

Reply:
allways show this error